donate.dd

```diff
@@ -0,0 +1,91 @@
+Ddoc
+
+$(D_S $(TITLE),
+
+$(H4 The $(LINK2 $(ROOT_DIR)foundation.html, D Language Foundation) is an organization devoted to advancing
+     open source technology related to the D programming language.
```

Comment:
Add this text: The Foundation is a $(HTTPS en.wikipedia.org/wiki/501(c)_organization, 501(c) non-profit public charity) registered in the state of Washington, USA.